app/components/search/minister-filter.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="au-c-card au-o-box au-o-box--small">
+  <div class="auk-form-group">
+    <Auk::Label for="ministerName">
+      Minister
+    </Auk::Label>
+    <AuInput
+      data-test-route-search-mandatee-input
+      @width="block"
+      @value={{this.mandateesBuffer}}
```

Comment:
This component doesn't seem to have a component class (JS file), so `this.mandateesBuffer` isn't referencing any variable and won't update the filters.